src/hevm/src/EVM/Types.hs

```diff
@@ -364,7 +364,7 @@ instance Show Addr where
   showsPrec _ addr next =
     let hex = showHex addr next
         str = replicate (40 - length hex) '0' ++ hex
-    in "0x" ++ toChecksumAddress str
+    in "0x" ++ toChecksumAddress str ++ drop 40 str
```

Comment:
what is this for?

Reply:
`next` is the continuation of things that need to be printed. Not keeping it means printing an address can eat up the rest of the things you wanna print